docs/architecture/cloud-native/implement-app.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Implementing a Cloud Native app
```

Comment:
Do we need this Section? All that this says is to download the app. I think we can move this to the "Intro" section and drop this .md file. Also, provide a Wiki link for the walkthrough.